tests/model_fields/test_uuid.py

```diff
@@ -37,6 +37,13 @@ def test_null_handling(self):
         loaded = NullableUUIDModel.objects.get()
         self.assertEqual(loaded.field, None)
 
+    def test_pk_validated(self):
+        with self.assertRaises(TypeError):
```

Comment:
It's helpful to use `self.assertRaisesMessage` to ensure this is the `TypeError` that we expected.